README.md

```diff
@@ -101,7 +103,7 @@ Then, execute `just bootstrap` to install the necessary Go packages
 ### .env Setup
 
 To avoid race conditions or having to wait for index creation, the tests require a project with at least one pod index
```

Comment:
nit: I think we can remove this whole first part since it's not longer true that we're avoiding waiting for indexes to create.